etcdserver/raft.go

```diff
@@ -198,6 +201,13 @@ func (r *raftNode) start(s *EtcdServer) {
 					return
 				}
 
+				// the leader can write to its disk in parallel with replicating to the followers and them
```

Comment:
this is directly copied from the thesis. :) so i do not really want to change.

opinion? @heyitsanthony